scripts/execution_layer.py

```diff
@@ -55,6 +55,29 @@ def parse_args():
         help="Print the operations, without running them",
     )
 
+    merge = subparsers.add_parser(
+        "merge",
+        help="Apply the changes made to FEATURE since it was cut, to BASE.",
+    )
+    merge.set_defaults(cmd="Merge", do=do_merge)
+    merge.add_argument("base", type=feature, help="The cut to merge into")
+    merge.add_argument("feature", type=feature, help="The cut to take from")
+    merge.add_argument(
+        "--dry-run",
+        action="store_true",
+        help="Print the patch file to apply to BASE without applying it.",
+    )
+    merge.add_argument(
+        "-f",
+        "--force",
+        action="store_true",
+        help=(
+            "Merge regardless of warnings of that the working directory is "
+            "not clean.  If the merge fails at this point, it may be "
```

Comment:
bit of word salad 😄 not sure if this should read "... regardless of warnings if the working directory is not clean"?

Reply:
😬 thanks for spotting!